<ui version="4.0" >
 <class>Catalog</class>
 <widget class="QMainWindow" name="Catalog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Catalog Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QTreeWidget" name="treeCatalog" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menuProject" >
    <property name="title" >
     <string>&amp;Project</string>
    </property>
    <addaction name="actionConfigure" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionFind_in_Files" />
    <addaction name="actionReplace_in_Files" />
    <addaction name="actionStop_Find" />
    <addaction name="separator" />
    <addaction name="actionReload" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuProject" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionReload" />
   <addaction name="actionConfigure" />
  </widget>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset>../images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionReload" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/reload.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="icon" >
    <iconset>../images/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configure...</string>
   </property>
  </action>
  <action name="actionFind_in_Files" >
   <property name="icon" >
    <iconset>../images/find.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find in Files...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace_in_Files" >
   <property name="icon" >
    <iconset>../images/replace.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Replace in Files...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop_Find" >
   <property name="icon" >
    <iconset>../images/stop.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Stop Find</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
